### DISCUSSION TITLE
I want transparent background instead of standard background of mantine theme

### DISCUSSION THREAD

**Discussion by ghost** [mantinedev/mantine#5262] [Q&A]
I'm a Fivem Frontend developer and today i used for the first time Mantine for create a Slider and a DateInput, all works good but when i set the body to none with this:

import { createContext, useContext, useState } from 'react';

import { useNuiEvent } from '@hooks/index';
import { debugData } from '@utils/index';

const VisibilityContext = createContext(null);

export const VisibilityProvider = ({ children }) => {
const [visible, setVisible] = useState(false);

useNuiEvent('SET_VISIBLE', ({ status }) => {
setVisible(status);

if (status) {
  document.body.style.display = 'block';
} else {
  document.body.style.display = 'none';
}
});

return (
<VisibilityContext.Provider value={{ visible, setVisible }}>
{visible && children}
</VisibilityContext.Provider>
);
};

export const useVisibility = () => useContext(VisibilityContext);

the background became dark instead of transparent, i check on browser styles and i saw that if i check on square next to this and disable it: --mantine-color-scheme: light dark; the background became transparent, how can i disable this in my code forever?

**Comment by tombohub** [comment]
can you please post formatted code
